samples/esb/esb_prx/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _esb_prx:
+
+Enhanced ShockBurst: Receiver
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows the Enhanced ShockBurst Receiver application.
+It includes the basic steps that are needed to transmit and receive packets using :ref:`ug_esb`.
+Follow the instructions and use the files from this page, and the :ref:`esb_ptx` sample.
+
+Requirements
+************
+
+.. note::
+   Follow the steps from and include the :ref:`esb_ptx` sample with this sample.
```

Comment:
```suggestion
   You also need to use the :ref:`esb_ptx` sample to test this sample.
   Follow the steps from either sample's documentation.
```
The steps are duplicated in each documentation, so following them from both seems like the wrong instruction here.

I am not sure the note is needed, though. For example, the [Thread: CoAP Client](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/coap_client/README.html#requirements) sample only has the following sentence under the platform table:

> You can use one or more of the development kits listed above as the Thread CoAP Client. You also need one or more compatible development kits programmed with the [Thread: CoAP Server](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/coap_server/README.html#coap-server-sample) sample.

Reply:
These are good points. I followed what we did when we split the [Gazell samples](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/gazell.html). I will think it over and see if it can be done differently.